src/util/device-actions/device-actions-service.ts

```diff
@@ -21,16 +21,15 @@ interface GetActionStatusParams {
 	extraOptions?: any;
 }
 
-const DEVICE_ACTIONS_API_VERSION = 'v1';
-
 export class DeviceActionsService {
 	private actionsEndpoint: string;
 
 	constructor(
 		deviceUrlsBase: string,
+		deviceActionsApiVersion: string,
```

Comment:
We can be a bit more precise with:
```suggestion
		deviceActionsApiVersion: 'v1' | 'v2',
```